DependencyInjection/LexikJWTAuthenticationExtension.php

```diff
@@ -120,6 +120,12 @@ public function load(array $configs, ContainerBuilder $container)
             $container
                 ->getDefinition('lexik_jwt_authentication.handler.authentication_success')
                 ->replaceArgument(2, new IteratorArgument($cookieProviders));
+
+            $container
+                ->getDefinition('lexik_jwt_authentication.generate_keypair_command')
+                ->replaceArgument(1, $config['secret_key'])
+                ->replaceArgument(2, $config['public_key'])
+                ->replaceArgument(3, $config['pass_phrase']);
```

Comment:
Try moving the calls outside of the if block they are currently wrapped in, it should go better :)
(I must admit I had to stare at it for a while...)

Reply:
Eagle eyes! 😄 

(Facepalming myself) 🤦 
It works, thanks a lot! 🙏